src/index.spec.ts

```diff
@@ -1,12 +1,227 @@
+import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
+enableFetchMocks();
 import {
   isInteger,
   toLowerCase,
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
 test('Testing time!', () => {
   console.log('welcome');
 });
+
+describe('isInteger test', () => {
+  it('should return true with positive integer', () => {
+    expect(isInteger(5)).toBeTruthy();
+  });
+  it('should return true with negative integer', () => {
+    expect(isInteger(-5)).toBeTruthy();
+  });
+  it('should return true with 0', () => {
+    expect(isInteger(0)).toBeTruthy();
+  });
+  it('should return true with -0', () => {
+    expect(isInteger(-0)).toBeTruthy();
+  });
+  it('should return false with decimal number', () => {
+    expect(isInteger(0.5)).toBeFalsy();
+  });
+  it('should return false with negative decimal number', () => {
+    expect(isInteger(-0.5)).toBeFalsy();
+  });
+});
+
+describe('toLowerCase test', () => {
+  it('should return abc with ABC', () => {
+    expect(toLowerCase('ABC')).toMatch(/abc/);
+  });
+  it('should return 23hF with 23HF', () => {
+    expect(toLowerCase('23HF')).toMatch(/23hf/);
+  });
+  it('should return 23hF with 23hf', () => {
+    expect(toLowerCase('23hf')).toMatch(/23hf/);
+  });
+  it('should return abc with abc', () => {
+    expect(toLowerCase('abc')).toMatch(/abc/);
+  });
+  it('should return Please provide a string with empty string', () => {
+    expect(toLowerCase('')).toBe('Please provide a string');
+  });
+});
+
+describe('removeDuplicatesFromArray test', () => {
+  it('should return [1] with [1]', () => {
+    expect(removeDuplicatesFromArray([1])).toEqual([1]);
+  });
+  it('should return [1,"a",3] with [1,"a",3]', () => {
+    expect(removeDuplicatesFromArray([1, 'a', 3])).toEqual([1, 'a', 3]);
+  });
+  it('should return [1,2,3] with [1,2,3,2]', () => {
+    expect(removeDuplicatesFromArray([1, 2, 3, 2])).toEqual([1, 2, 3]);
+  });
+  it('should return Error with any type diferent to string[] or number[]', () => {
+    const testFunction = () => {
+      removeDuplicatesFromArray({} as any);
+    };
```

Comment:
Why are you creating a wrapper of your testing function?

Avoid this, don't create wrappers of your testing functions. And avoid use any as well